service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskExecutor.java

```diff
@@ -54,7 +54,16 @@ public void execute(GetGpcDocumentTaskDefinition taskDefinition) {
             ehrExtractStatus = handleValidGpcDocument(response, taskDefinition);
         } catch (GpConnectException e) {
             LOGGER.warn("Binary request returned an unexpected response", e);
-            ehrExtractStatus = getAbsentAttachmentTaskExecutor.handleAbsentAttachment(taskDefinition);
+
+            var exceptionDisplay = e.getOperationOutcome()
```

Comment:
Will probably need to make this code more defensive, dealing with cases:
- NULL operation outcome
- no issue
- no coding

etc.